docs/src/timearray.md

```diff
@@ -48,8 +48,18 @@ where `n` enumerates from 1.
 ## `meta`
 
 The `meta` field defaults to holding `nothing`, which is represented by
-type `Void`. This default is designed to allow programmers to ignore
+type `Nothing`. This default is designed to allow programmers to ignore
 this field. For those who wish to utilize this field, `meta` can hold
 common types such as `String` or more elaborate user-defined types. One
 might want to assign a name to an object that is immutable versus
 relying on variable bindings outside of the object's type fields.
+
+## Fields getter functions
+
+There are four field getter functions exported.
+They are named as same as the field names.
```

Comment:
Field names are effectively an implementation detail, so I don't know that it's worth mentioning that. Good to document these functions though.

Reply:
I think the field name is revealed to user already here:
http://juliastats.github.io/TimeSeries.jl/latest/timearray.html

So I add that section.